src/index.css

```diff
@@ -35,18 +35,24 @@ body {
 	font-family: var( --font-san-serif );
 	padding: 0.75rem 0.5rem;
 	width: 100%;
-	border-radius: 4px;
+	border-radius: 2px;
 	background-color: var( --color-white );
-	border: 2px solid var( --color-border-dark );
+	border: 1px solid var( --color-border-dark );
 	box-sizing: border-box;
 }
 
+.root input[type='text']:hover,
+.root input[type='search']:hover,
+.root input[type='email']:hover {
+	outline: 1px solid var( --color-border-dark );
```

Comment:
> Would it be possible to remove that and just use the hover state for search bar itself or is there an accessibility-related reason to have both?

Hmm, that's a good question! 🤔 The border on hover definitely looks goofy -- we should change that for sure! There's not really accessibility concerns per se because we actually remove that button from the accessibility tree (it's just for the mouse), but the challenge is that there's not really a good way to apply the hover style to the whole input when you hover over the button (this is a side effect of how you have to absolutely position buttons over text inputs).

As a middle ground, how would you feel if we applied to the search button the basic hover=neutral-extra-light, active=neutral-light that we do to the rest of our other clear buttons?

Reply:
Btw, I've pushed that up if you want to take a look and play around with it!

Reply:
> Hmm, that's a good question! 🤔 The border on hover definitely looks goofy -- we should change that for sure!

Gahhh, you're going to _hate_ me, but even though I tested `--color-neutral-extra-light` as the hover background _before_ suggesting it to you, I'm now realizing that it kind of makes the search bar appear disabled. Especially if you look at it on the feature selector. 

Context/apology - a lot of search bars have hover states, but they're usually more subtle and in the form of a drop shadow. Since the WordPress design library isn't big on drop shadows for components, I decided to try using border to indicate hover, which ended up looking goofy. Then I thought using background color would be better. In hindsight, the WordPress design library doesn't have a hover state for the search bar and I guess this could be the reason why 😅 

As terrible as it sounds, I'm leaning toward just taking out the hover state for the search bar, since hover seems to be the most beneficial on clickable components like buttons, dropdowns, etc. I'm so sorry for steering you in the wrong direction 🙇 

> As a middle ground, how would you feel if we applied to the search button the basic hover=neutral-extra-light, active=neutral-light that we do to the rest of our other clear buttons?

Wow, I like this _a lot_! It makes it way clearer that it's a button that can be clicked and I think we should keep that regardless of the search bar hover style 👍